### ISSUE TITLE
Searching with invalid key doesn't abort search

### ISSUE THREAD

**Issue by norgio** [cambragol/advent-of-the-trinity#117]
Weird instance where when searching: instead of saying "INVALID DIRECTION" or something to that effect, entering character keys instead of arrow keys just scrolls those character keys rather than aborting.

![image](https://user-images.githubusercontent.com/74744465/101426913-b65fd500-38b2-11eb-804e-fba3b3eec2ce.png)

**Comment by cambragol** [comment]
Hmmm..is this different from default DOS?

**Comment by norgio** [comment]
it would probably have to be different from default dos because default DOS - if i'm remembering my playthroughs correctly - doesn't allow you to search in a direction: search command searches the square you are standing on.

**Comment by cambragol** [comment]
Of course.  However, I think I am using the default directional command function.

I will check that, but likely a similar case can be observed in other places where you can choose directions.  Like casting certain spells I think....

**Comment by norgio** [comment]
very interesting.  similar but slightly different behavior with the attack command. 

![image](https://user-images.githubusercontent.com/74744465/101437073-45291d80-38c4-11eb-94b5-bfd4bdf2a0dc.png)

**Comment by cambragol** [comment]
How about default?  I can't check just at this moment.

**Comment by norgio** [comment]
here's a screen capture of U4 when attempting to attack without a direction

![image](https://user-images.githubusercontent.com/74744465/101437247-99340200-38c4-11eb-8679-f0e8ff3b1c4d.png)

**Comment by cambragol** [comment]
So, looks like default behaviour. However, looking through the code I did add stuff to the AskDir function, to allow for Southwest, Northwest directions etc. I also added a timeout, like in Apple II (and C64?)
Other than that it is default behaviour.

So I think we could close this...

However, I looked at default Apple II behavior, and we see this:

<img width="582" alt="Screen Shot 2020-12-08 at 14 43 37" src="https://user-images.githubusercontent.com/5019739/101445043-19f60b00-3964-11eb-89b7-8380cc922410.png">

I tweaked Trinity to do that, and saved 3 lines of code! Yes!

![Ultima IV Trinity Project Build Environment 2020-12-08 at 2 44 03 PM](https://user-images.githubusercontent.com/5019739/101445098-372ad980-3964-11eb-994f-d267f578d361.png)

So I think I want to go with this. Nice!
